istio-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: istio-operator
-version: 0.0.5
+version: 0.0.7
```

Comment:
Why not 0.0.6? :)

Reply:
Because I already used that version for an internal dev release. :)

Reply:
Okay :)